moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -53,6 +53,7 @@ namespace core
  * it is difficult to choose a jump_threshold parameter that effectively separates
  * valid paths from paths with large joint space jumps. */
 static const std::size_t MIN_STEPS_FOR_JUMP_THRESH = 10;
+std::string RobotState::name_ = "robot_state";
```

Comment:
If using this one (which I don't prefer), please use `static const`.

Reply:
This refers to the same static member variable. An easy test to demonstrate this is to make one const and not the other.

Reply:
I have made it const